submissions/Lu-sure/dom-api/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&display=swap" rel="stylesheet">
+        <script type="module" src="index.js"></script>
+        <title>BBT</title>
+    </head>
+    <body>
+        <div class="wrapper">
+
+            <aside class="side" id="side">
+            </aside>
+
+            <main>
+                <div class="side-button-wrapper">
```

Comment:
Oh, I understand. You mean using an <aside> tag out of main would be more semantic?

Reply:
The side button is in main block. It appears when the screen width is less than 481px, it is a burger-like menu button, which opens side-menu. Please, tell me, how it would be write to name this class

Reply:
Renamed, and moved out of `main` tag.